src/components/rocket.js

```diff
@@ -0,0 +1,45 @@
+import { React, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getInformation, setReservationState } from '../redux/rocket/rocket';
+
+const RocketComponent = () => {
+  const dispatch = useDispatch();
+  const RocketData = useSelector((state) => state.rocketReducer);
+  useEffect(() => {
+    dispatch(getInformation([]));
+  }, []);
+
+  const handleRocketStateClick = (key, status) => {
+    if (status) {
+      dispatch(setReservationState({
+        id: key,
+        value: false,
+      }));
+    } else {
+      dispatch(setReservationState({
+        id: key,
+        value: true,
+      }));
+    }
```

Comment:
- [ ] All logic should be in the reducers, then dispatched on react view files. Please, the status change and this kind of thing is something you could add on the reducers, please work on it. 🐶